resources/content/schema/base/service.json

```diff
@@ -68,6 +68,11 @@
             "type": "array[publicEndpoint]",
             "required": false,
             "nullable": true
+        },
+        "dnsToken": {
+            "type": "string",
+            "required": true,
```

Comment:
this field shouldn't be required as we generate it internally. This setting might break service.update for the service on upgraded setups